core-bundle/src/Cron/PruneExpiredDataCron.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Cron;
+
+use Contao\Config;
+use Contao\CoreBundle\Framework\ContaoFramework;
+use Contao\CoreBundle\ServiceAnnotation\CronJob;
+use Doctrine\DBAL\Connection;
+
+class PruneExpiredDataCron
+{
+    /**
+     * @var ContaoFramework
+     */
+    private $framework;
+
+    /**
+     * @var Connection
+     */
+    private $connection;
+
+    public function __construct(ContaoFramework $framework, Connection $connection)
+    {
+        $this->framework = $framework;
+        $this->connection = $connection;
+    }
+
+    /**
+     * @CronJob("hourly")
+     */
+    public function onHourly(): void
+    {
+        $this->framework->initialize();
+
+        $undoPeriod = (int) $this->framework->getAdapter(Config::class)->get('undoPeriod');
```

Comment:
Instead of using `$this->framework->getAdapter(Config::class)` three times, why don't we use the following?

```php
/** @var Config $config */
$config = $this->framework->getAdapter(Config::class)
```

This will also make PhpStorm understand the `->get()` call.

Reply:
I don't mind - CS :)